### ISSUE TITLE
Docstring of `intersection_all` incorrect (graph attrs are copied)

### ISSUE THREAD

**Issue by eriknw** [networkx/networkx#6750]
### Current Behavior

Currently, `intersection_all` copies the `.graph` attributes to the result, and attributes from the later graphs have higher precedense. However, the docstring says:
```
Attributes from the graph, nodes, and edges are not copied to the new graph.
```
### Expected Behavior

For the docstring to match behavior or behavior to match docstring.

### Steps to Reproduce

```python
In [1]: import networkx as nx

In [2]: G = nx.Graph()

In [3]: H = nx.Graph()

In [4]: G.graph[1] = 1

In [5]: G.graph[2] = 2

In [6]: H.graph[2] = 20

In [7]: H.graph[3] = 30

In [8]: R = nx.intersection_all([G, H])

In [9]: R.graph
Out[9]: {1: 1, 2: 20, 3: 30}
```

### Environment

Python version: 3.11
NetworkX version: 3.1

**Comment by eriknw** [comment]
Also, same issue for `intersection`.

**Comment by dschult** [comment]
Fixed by #6744
